lessons/11_Lesson11_vortexSourcePanelMethod.ipynb

```diff
@@ -537,14 +545,14 @@
     "    \n",
     "    Returns\n",
     "    -------\n",
-    "    Contribution from panel at point (x, y).\n",
+    "    Contribution from the panel at a given point (x, y).\n",
     "    \"\"\"\n",
-    "    def func(s):\n",
+    "    def integrand(s):\n",
     "        return ( ((x - (panel.xa - numpy.sin(panel.beta)*s))*dxdk\n",
```

Comment:
Lines should end with a mathematical operator, it will help the notebook formatter line up the parentheses in a nicer way.